.github/workflows/sanitize-alert-schedule.yml

```diff
@@ -42,7 +42,11 @@ jobs:
         uses: ./.github/actions/sanitize
         with:
           lint: true
-          test: true
+          test-engine: true
+          test-grafbase: true
+          test-grafbase-gateway: true
```

Comment:
We need a consistent way of naming things.

Some options:
1. engine, grafbase, gateway
2. engine, cli, gateway
3. grafbase-engine, grafbase-cli, grafbase-gateway